micrometer-observation/src/main/java/io/micrometer/observation/Observation.java

```diff
@@ -59,19 +59,20 @@ public interface Observation extends ObservationView {
      * @return started observation
      */
     static Observation start(String name, ObservationRegistry registry) {
-        return start(name, null, registry);
+        return start(name, () -> null, registry);
```

Comment:
Should we provide a null context supplier so users can do something like this: `start(name, Context.NULL, registry)`?

Though if they want to pass null, they can just call the overload where this param is missing.

Reply:
I tried adding static supplier field to `Context`, but it doesn't really save anything compared to using `Context::new` instead, so I think I'll leave it out until/unless we get feedback from users. I'll switch these `() -> null` to `Context::new` instead.